language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/createvar/ErrorHandleOutsideCodeAction.java

```diff
@@ -103,7 +103,7 @@ public List<CodeAction> getCodeActions(Diagnostic diagnostic,
                 positionDetails.matchedNode(), context));
         edits.addAll(importsAcceptor.getNewImportTextEdits());
 
-        int renamePosition = modifiedTextEdits.renamePositions.get(0) - UNION_ERROR_CHAR_OFFSET;
+        int renamePosition = modifiedTextEdits.renamePositions.get(0);
```

Comment:
Since we are no longer using `UNION_ERROR_CHAR_OFFSET`, we can remove the constant created in line number 52.

Reply:
+1